docs/pages/api-docs/button.js

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
 import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
-import markdown from './button.md';
+import { prepareMarkdown } from 'docs/src/modules/utils/parseMarkdown';
 
-export default function Page() {
-  return <MarkdownDocs markdown={markdown} />;
+const pageFilename = 'api/button';
+const requireRaw = require.context('!raw-loader!./', false, /button\.md$/);
```

Comment:
This is matching `docs/pages/api-docs/toggle-button.md` rather than `docs/pages/api-docs/button.md`